config.json.sample

```diff
@@ -20,12 +19,12 @@
     	"S3" : {
         }, 
 
-        "OneClick" : {
+        "ThirdPartyProvider" : {
```

Comment:
OK, I'll change it back.  I thought you wanted to take the OneClick-specific stuff out of the config file and into the wiki.